.github/workflows/build.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           pip install pytest mock
           pip install jupyter-client ipykernel
           python -m ipykernel install --user --name python3
+          pip install -e ".[local-jupyter-exec]"
```

Comment:
I'd like to install optional dependencies only in a subset of environments, to make sure the minimal installation is usable. Lines 45-47 are supposed to be moved to a conditional step.

Reply:
Do you want me to add a new workflow for this? Or how would you like to manage it?